source/slang/slang-compiler.h

```diff
@@ -2089,6 +2109,21 @@ SLANG_FORCE_INLINE EndToEndCompileRequest* asInternal(SlangCompileRequest* reque
     return reinterpret_cast<EndToEndCompileRequest*>(request);
 }
 
+class ListBlob : public ISlangBlob, public RefObject
```

Comment:
Unrelated to this change: it would be a nice cleanup at some point in the future if we made `StringRepresentation` directly conform to `ISlangBlob` so that we could return them directly to the users without constructing a wrapper object.

The same thing *could* apply to `List<T>` if we decided to implement it with the same copy-on-write approach that we are currently using for `String`. I know that there are reasons why the C++ stdlib doesn't allow copy-on-write for stuff like `vector` (you need to make `operator[]` return a proxy if you want copy-on-write), so we may never go there, but seeing wrapper types like this makes me wonder if we could ever have a `ListRepresentation` object type that conforms to `ISlangBlob`.